# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-nios2.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

  MODPOST modules-only.symvers
arch/nios2/boot/Makefile:24: FORCE prerequisite is missing
ERROR: modpost: "__mulsi3" [net/vmw_vsock/vsock.ko] undefined!
...
WARNING: modpost: suppressed 1503 unresolved symbol warnings because there were too many)
make[1]: *** [scripts/Makefile.modpost:134: modules-only.symvers] Error 1

--- linux-5.15/arch/nios2/kernel/nios2_ksyms.c.vanilla	2021-12-30 13:50:48.173708808 +0100
+++ linux-5.15/arch/nios2/kernel/nios2_ksyms.c	2021-12-30 13:59:41.320687550 +0100
@@ -32,6 +32,8 @@
 
 DECLARE_EXPORT(__gcc_bcmp);
 DECLARE_EXPORT(__divsi3);
+DECLARE_EXPORT(__divsi3_table);
+DECLARE_EXPORT(__mulsi3);
 DECLARE_EXPORT(__moddi3);
 DECLARE_EXPORT(__modsi3);
 DECLARE_EXPORT(__udivmoddi4);
